gulpfile.js

```diff
@@ -60,7 +60,7 @@ gulp.task('clean', function() {
 gulp.task('copy-static', ['copy-html', 'copy-assets', 'copy-phaser']);
 
 gulp.task('copy-html', function() {
-  gulp.src([paths.base + '/index.html', ])
+  gulp.src([paths.base + '/*.{html,js,json}'])
```

Comment:
This is a minor comment. Not sure that copying all HTML and JSON files is preferred here since we don't expect creating many of those. For example, if we create test.html to test some alternate feature, this should probably not be copied over.

If copying more than HTML, perhaps this task should be renamed from `copy-html` for clarity.